src/cbapi/psc/livequery/models/device_summary.yaml

```diff
@@ -0,0 +1,37 @@
+type: object
+required: [] # TODO(ww): Find out which result fields are required
+properties:
+  id:
+    type: string
+    description: The result's unique ID
+  total_results:
+    type: integer
+    format: int32 # NOTE(ww): This is a guess
+    description: Number of results returned for this particular device
+  device_id:
+    type: integer
+    format: int32 # NOTE(ww): This is a guess
+    description: The device's unique ID
+  device_name:
+    type: string
+    description: The device's name
+  time_received:
```

Comment:
Would it be possible to include a format indicating the time is ISO format

Reply:
Yes.